src/dms-preview/azext_dms/custom.py

```diff
@@ -59,27 +65,21 @@ def create_or_update_project(
         # If not an extension scenario, run CLI core method
         # TODO: We currently don't have any CLI core code to perform any validations
         # because of this we need to raise the error here.
-        try:
-            # TODO: Remove this check after validations are added to core
-            if source_platform != "sql" or target_platform != "sqldb":
-                raise CLIError("The provided source-platform, target-platform combination is not appropriate. \n\
+
+        # TODO: Remove this check after validations are added to core
+        if source_platform != "sql" or target_platform != "sqldb":
+            raise CLIError("The provided source-platform, target-platform combination is not appropriate. \n\
 Please refer to the help file 'az dms project create -h' for the supported scenarios.")
 
-            core_res = core_create_or_update_project(
-                client,
-                project_name,
-                service_name,
-                resource_group_name,
-                location,
-                source_platform,
-                target_platform,
-                tags)
-        except:
```

Comment:
In this way, how do you validate your command now?

Reply:
I believe you are referring the fact that the core doesn't validate the type of project created. The core only handles the sql to sql scenario, this extension handles all the rest. What needs to happen, since these other scenarios are coming off of public preview, is we will migrate these scenarios out of the extension and over to the core.

But right now, the type of project allowed is handled in the extension_handles_scenario method.